### PR TITLE
Whitespace cleanup in Windows implementation

### PR DESCRIPTION
**Pull request by DJm00n**
For better diffs in future.

### CODE REVIEW

File: hidapi/hidapi.h [hunk 1]
```diff
@@ -30,11 +30,11 @@
 #include <wchar.h>
 
 #ifdef _WIN32
-      #define HID_API_EXPORT __declspec(dllexport)
-      #define HID_API_CALL
+	  #define HID_API_EXPORT __declspec(dllexport)
```
Comment (Youw) [inline hidapi/hidapi.h:33]: you've replaced some of the spaces with tabs, but not all of them

consistency (even across small block of code) is better than "some general rules"

File: windows/hid.c [hunk 2]
```diff
@@ -417,7 +417,7 @@ struct hid_device_info HID_API_EXPORT * HID_API_CALL hid_enumerate(unsigned shor
 		/* Check the VID/PID to see if we should add this
 		   device to the enumeration list. */
 		if ((vendor_id == 0x0 || attrib.VendorID == vendor_id) &&
-		    (product_id == 0x0 || attrib.ProductID == product_id)) {
+			(product_id == 0x0 || attrib.ProductID == product_id)) {
```
Comment (Youw) [inline windows/hid.c:420]: IMHO "before" is better then "after"
1) Now part the condition is aligned with the code, which a con from readability POV
2) Before, the start of the condition in each line was aligned which is proc from readability POV

File: windows/hid.c [hunk 3]
```diff
@@ -552,12 +552,12 @@ HID_API_EXPORT hid_device * HID_API_CALL hid_open(unsigned short vendor_id, unsi
...
 	const char *path_to_open = NULL;
 	hid_device *handle = NULL;
-	
+
 	devs = hid_enumerate(vendor_id, product_id);
 	cur_dev = devs;
 	while (cur_dev) {
 		if (cur_dev->vendor_id == vendor_id &&
-		    cur_dev->product_id == product_id) {
+			cur_dev->product_id == product_id) {
```
Comment (Youw) [inline windows/hid.c:560]: ditto

**Review by Youw — COMMENTED** [review]
Please see related discussion:
https://github.com/libusb/hidapi/pull/215

File: windows/hid.c (hunk 2 above)
Comment (DJm00n) [reply windows/hid.c:420]: Alright. Anyways current state when we have mixed tabs and spaces is a mess.

Can we stick to spaces-only (I prefer that) instead?

Comment (Youw) [reply windows/hid.c:420]: historically HIDAPI is a tabs-only (except a few exceptions)

**Comment by DJm00n** [comment]
@Youw I have removed spaces-to-tabs conversion. Now removing extra spaces only.

**Review by Youw — APPROVED** [review]

**Pull request merged**